packages/core/src/components/dyte-virtualized-list/dyte-virtualized-participant-list.tsx

```diff
@@ -0,0 +1,183 @@
+import { Component, h, Prop, State, Element, Watch } from '@stencil/core';
```

Comment:
For later but could we club the virtualisation logic, rn it is duplicated across chat and participants

Reply:
Yeah.  Plan is to virtualise chat as well with a similar logic rather than the current paginated view. However it would be a breaking change since the paginated chat component asks for a page size. Will raise a separate PR for that. I will write a virtualisation logic that can take care of dynamic height and widths. Current virtualisation logic only supports lists that have all the members of the same fixed height.

Reply:
Created a ticket. Will do this later.